### PR TITLE
Added voice to the friendly skulls.

### PR DESCRIPTION
**Pull request by Fedic**

### CODE REVIEW

File: Source/Redemption/Private/SkullNPC.cpp [hunk 1]
```diff
@@ -73,3 +80,20 @@ void ASkullNPC::Tick(float DeltaTime)
...
+    if (CurrentSoundIndex < TotalSounds)
+    {
+        if (XyloSounds != nullptr)
+        {
+            float RandomPitch = FMath::RandRange(0.5f, 1.5f);
+            UGameplayStatics::PlaySoundAtLocation(this, XyloSounds, GetActorLocation(), FRotator::ZeroRotator, 1.0f, RandomPitch);
+        }
+
+        // Schedule the next sound playback after a delay
+        GetWorldTimerManager().SetTimer(SoundTimerHandle, [this, CurrentSoundIndex, TotalSounds]()
```
Comment (Entwistle-905) [inline Source/Redemption/Private/SkullNPC.cpp:94]: how does this work?

**Review by Entwistle-905 — APPROVED** [review]
Just solve the merge conflict

File: Source/Redemption/Private/SkullNPC.cpp (hunk 1 above)
Comment (Fedic) [reply Source/Redemption/Private/SkullNPC.cpp:94]: This part sets up the next sound playback after a 0.2-second delay using a timer.It calls the PlayNextSound function with the next sound index after that delay.

Comment (Fedic) [reply Source/Redemption/Private/SkullNPC.cpp:94]: Also I dont understand why theres a merge conflict i should have been the only one changing the bp. I don't know how to fix that

**Review by mac2648 — APPROVED** [review]

File: Source/Redemption/Private/SkullNPC.cpp (hunk 1 above)
Comment (mac2648) [reply Source/Redemption/Private/SkullNPC.cpp:94]: the last time the BP_Talkingskull was changed was 10 days ago, I can only imagine that for some reason you you pulled it you did not receive it, or that the creation of the branch happened before the blueprint was merged to main

Comment (mac2648) [reply Source/Redemption/Private/SkullNPC.cpp:94]: I can help you with solving the conflict

**Pull request merged**